parts/k8s/windowskubeletfunc.ps1

```diff
@@ -316,9 +316,14 @@ New-NSSMService {
         [string]
         [Parameter(Mandatory = $true)]
         $KubeProxyStartFile
+        [Parameter(Mandatory = $false)][string]
+        $ContainerRuntime = "docker"
     )
 
     $kubeletDependOnServices = "docker"
+    if ($ContainerRuntime -eq "containerd") {

```

Comment:
You can just set `kubeletDependOnServices = $ContainerRuntime` and eliminate the conditional here.